index.php

```diff
@@ -72,7 +74,10 @@
 $version = isset($_GET['version']) ? (string)$_GET['version'] : null;
 $isSparkle = isset($_GET['sparkle']) ? true : false;
 $isFileProvider = isset($_GET['fileprovider']) ? true : false;
-$channel = isset($_GET['channel']) ? (string)$_GET['channel'] : 'stable';
+// due to a bug in an old version, the channels were translated. we need to catch them again
+$channel = isset($_GET['channel']) && in_array($_GET['channel'], $allowedChannels, true)
+	? $_GET['channel']
```

Comment:
might want to include the string cast in there again

```suggestion
$channel = isset($_GET['channel']) && in_array((string)$_GET['channel'], $allowedChannels, true)
	? (string)$_GET['channel']
```